lego/apps/events/tasks.py

```diff
@@ -226,9 +236,19 @@ def stripe_webhook_event(self, event_id, event_type, logger_context=None):
             raise WebhookDidNotFindRegistration(event_id, metadata)
         registration.charge_id = serializer.data["id"]
         registration.charge_amount = serializer.data["amount"]
-        registration.charge_amount_refunded = serializer.data["amount_refunded"]
+        registration.charge_amount_refunded = (
```

Comment:
Hmm :thinking:

Reply:
This feels bad

Reply:
Maybe make the field nullable if stripe returns null/undefined?